src/sokoban.hpp

```diff
@@ -1,22 +1,27 @@
 #ifndef __SOKOBAN_H__
 #define __SOKOBAN_H__
 
+#include <iostream>
+#include <stdexcept>
 #include <string>
 #include <vector>
 
 class Sokoban {
     std::vector<std::vector<std::string>> levels;
     std::vector<std::string> _board;
     unsigned int current_level;
-    unsigned int px;
     unsigned int py;
+    unsigned int px;
     void locate_player();
-	
+    bool make_move(int dy, int dx);
```

Comment:
We could probably move the properties above the methods and bunch methods and properties. Really minor.